src/chi-vue/src/constants/types.ts

```diff
@@ -184,6 +184,25 @@ export type PaginationSizes = typeof PAGINATION_SIZES[number];
 //#endregion
 
 //#region Save View
+export interface SaveViewConfig {
+  active: boolean;
+  results?: string;
```

Comment:
Previously the `results` [was marked with non-null operator](https://github.com/CenturyLink/Chi/pull/573/files#diff-b6d44b1f144cbcdda5615a108b5be0761def07770f432bbead3a0bf0a1ae65acL21)  

Here I see the `results` can be undefined, so probably it would be a good idea either:
- change it to non-null and non-undefined (as it was before)
- or add a default value for the `results` to [src/chi-vue/src/components/data-table-save-view/default-config.ts](https://github.com/CenturyLink/Chi/pull/573/files#diff-e98dd95201caf3b0aab235db741a1b9931b2f15ad9585da731c7b7288e037152R3)

Reply:
Yes, good catch here so I guess it's better to keep the same logic as it was before with the non-null, making this property mandatory. What do you think @jllr @SergioQCL?